src/RevCPU.cc

```diff
@@ -2628,17 +2631,17 @@ void RevCPU::SetupArgs(uint32_t ThreadIDToSetup, RevFeature* feature){
 // if it does and there is work to assign (ie. ThreadQueue is not empty)
 // assign it and enable the processor if not already enabled.
 void RevCPU::UpdateThreadAssignments(uint32_t ProcID){
+  // print the thread queue
   // Get utilization info
   double Util = Procs[ProcID]->GetHartUtilization();
-  if( Util > 0.0 ){
-    output.verbose(CALL_INFO, 10, 0, "Core %" PRIu32 " utilization: %.2f%%\n", ProcID, Util);
-  }
+  //if( Util > 0.0 ){
+    output.verbose(CALL_INFO, 11, 0, "Core %" PRIu32 " utilization: %.2f%%\n", ProcID, Util);
+  //}
```

Comment:
Is this intended to be checked-in as part of the PR, or is this leftover debugging output?

Reply:
All `output.verbose` lines are not debugging... If you ever see `std::cout` in a PR then definitely point that out. Anything using the SST::Output object is meant to be there. I'll remove the surrounding `if` comments though

Reply:
It was the comments I was referring to as "debugging code", like you were turning it always on for debugging.